sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -716,4 +716,8 @@ class DDLSuite extends QueryTest with SharedSQLContext with BeforeAndAfterEach {
     }
   }
 
+  test("set / set -v") {
+    checkExistence(sql("set"), true, "env:", "system:")
```

Comment:
Found out that current test cases SQLQuerySuite and HiveQuerySuite have intensive test coverage for SET command.